config/initializers/openstax_utilities.rb

```diff
@@ -9,7 +9,7 @@
       next
     end
 
-    next if !Rails.application.is_real_production? || current_user.is_administrator?
+    next if !is_real_production? || current_user.is_administrator?
```

Comment:
is this working because the initilizer is named is_real_production or can you always access methods from other config files?

Reply:
The initializer just defines a global `is_real_production?` method.

Reply:
Huh... how is it reading `secrets` though? hmm
Also it's only used in 1 place, I might as well move it there